css/types/basic-shape.json

```diff
@@ -270,30 +270,67 @@
-                    "value_to_set": "true"
-                  }
-                ]
-              },
-              "firefox_android": {
-                "version_added": "63",
-                "partial_implementation": true,
-                "notes": "Only supported on the <code>clip-path</code> and <code>offset-path</code> properties.",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "layout.css.clip-path-path.enabled",
-                    "value_to_set": "true"
-                  }
-                ]
-              },
+              "firefox": [
+                {
+                  "version_added": "63",
+                  "partial_implementation": true,
+                  "notes": [
+                    "From version 97, the <code>d</code> SVG presentation attribute supports <code>path()</code>.",
+                    "From version 71, the <code>clip-path</code> property supports <code>path()</code>.",
+                    "Before version 71, only the <code>offset-path</code> property supports <code>path()</code>."
+                  ]
+                },
+                {
+                  "version_added": "91",
+                  "version_removed": "97",
+                  "partial_implementation": true,
+                  "flags": [
+                    {
+                      "type": "preference",
+                      "name": "layout.css.d-property.enabled",
+                      "value_to_set": "true"
+                    }
+                  ],
+                  "notes": "The <code>d</code> SVG presentation attribute, <code>clip-path</code> CSS property, and <code>offset-path</code> CSS property support <code>path()</code>."
+                },
+                {
+                  "version_added": "64",
+                  "version_removed": "71",
+                  "partial_implementation": true,
+                  "flags": [
+                    {
+                      "type": "preference",
+                      "name": "layout.css.clip-path-path.enabled",
+                      "value_to_set": "true"
+                    }
+                  ],
+                  "notes": "The <code>clip-path</code> and <code>offset-path</code> properties support <code>path()</code>."
+                }
+              ],
+              "firefox_android": [
+                {
+                  "version_added": "63",
+                  "partial_implementation": true,
+                  "notes": [
+                    "From version 97, the <code>d</code> SVG presentation attribute supports <code>path()</code>.",
+                    "From version 79, the <code>clip-path</code> property supports <code>path()</code>.",
+                    "Before version 79, only the <code>offset-path</code> property supports <code>path()</code>."
+                  ]
+                },
+                {
+                  "version_added": "64",
```

Comment:
Note, I copied the pattern for FF android too. Note though that only the earlier preference applies - after v79 the settings UI disappeared.